core/src/main/resources/error/error-classes.json

```diff
@@ -14,13 +34,25 @@
     "message" : [ "Found duplicate keys '%s'" ],
     "sqlState" : "23000"
   },
+  "EMPTY_OPTION" : {
+    "message" : [ "Option `%s` can not be empty." ],
+    "sqlState" : "08006"
+  },
   "FAILED_RENAME_PATH" : {
     "message" : [ "Failed to rename %s to %s as destination already exists" ],
     "sqlState" : "22023"
   },
   "FAILED_SET_ORIGINAL_PERMISSION_BACK" : {
     "message" : [ "Failed to set original permission %s back to the created path: %s. Exception: %s" ]
   },
+  "FAILED_TO_MERGE_IN_COMPATIBLE_SCHEMAS" : {
+    "message" : [ "Failed to merge incompatible schemas %s and %s" ],
+    "sqlState" : "42000"
+  },
+  "FOUND_DUPLICATE_FIELD_IN_CASE_INSENSITIVE_MODE" : {
+    "message" : [ "%s" ],
```

Comment:
We should put the base message in here and provide only the parameters. In this case, we could make the message something like `Found duplicate field(s) %s: %s in case-insensitive mode` and pass in `requiredFieldName` and `matchedOrcFields`.